mos-platform/mega65/dma.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2023 LLVM-MOS Project
+// Licensed under the Apache License, Version 2.0 with LLVM Exceptions.
+// See https://github.com/llvm-mos/llvm-mos-sdk/blob/main/LICENSE for license
+// information.
+//
+// C++ support for MEGA65 DMA operations.
+//
+
+#pragma once
```

Comment:
Prefer include guards for nit-picky compiler-person reasons.

Reply:
fixed.

Reply:
added.